app/assets/stylesheets/components/_filter-summary.scss

```diff
@@ -0,0 +1,60 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-summary__remove-filters {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.app-c-filter-summary__remove-filter {
+  border: 1px solid $govuk-border-colour;
+  border-radius: govuk-em(5, 16px);
+  padding: govuk-spacing(2);
+  margin: 0 govuk-spacing(2) govuk-spacing(1) 0;
```

Comment:
Now that we're using `flex`, is it worth using a `gap: govuk-spacing(1)` on the parent instead? Reckon that'll make the spacing more consistent:
<img width="419" alt="image" src="https://github.com/user-attachments/assets/2c5a321f-f5df-49b6-8967-607da12a88c0">